src/styles/Utils/reset.scss

```diff
@@ -0,0 +1,27 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
let's move this one to separate normalize file
*it's not a reset property

Reply:
+ fixed it